### DISCUSSION TITLE
Domain Name

### DISCUSSION THREAD

**Discussion by januridp** [dzikirqu/dzikirqu-android#13] [Ideas]
Assalamualaikum, langsung aja. boleh saya kontribusi domain name dzikirqu.id ? Kebetulan masih tersedia di internet alias belum ada yang pakai. atau ada nama domain lain yang ingin di pakai?

**Comment by ImBIOS** [comment]
Wa'alaikumussalam, thanks untuk tawarannya mas. Currently, we have dzikirqu.com.
@wagyufari, what do you think about it?

**Reply by januridp** [reply]
oke deh lanjutkan ya 😎

**Comment by wagyufari** [comment]
Wa'alaikumsalam warahmatullah wabarakatuh, qadarullah akhi, kami sudah mempunyai domain dengan nama dzikirqu.com dan insyaAllah sedang dalam proses pembuatan landing page.

**Reply by januridp** [reply]
oke deh mantap 😎
